src/Components/Employee/ListWorkedHours/index.js

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTimesheets } from 'redux/timeSheets/thunks';
+import { Button, Modal, Spinner, Table } from 'Components/Shared';
+import styles from './list.module.css';
+
+const ListWorkedHours = () => {
```

Comment:
```suggestion
const TimeSheets = () => {
```
(rename also the folder)